ports/webcraft/portfile.cmake

```
@@ -0,0 +1,27 @@
+# ports/webcraft/portfile.cmake
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
```

```suggestion
if(VCPKG_TARGET_IS_WINDOWS)
    vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
endif()
```
assuming that dllexport declarations are the only problem.